go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	github.com/rudderlabs/analytics-go v3.3.3+incompatible
 	github.com/rudderlabs/bing-ads-go-sdk v0.2.3
 	github.com/rudderlabs/compose-test v0.1.3
-	github.com/rudderlabs/rudder-go-kit v0.48.0
+	github.com/rudderlabs/rudder-go-kit v0.48.1-0.20250327144458-e24b01c6209e
```

Comment:
let's use the [0.48.1 release](https://github.com/rudderlabs/rudder-go-kit/pull/764)